.circleci/config.yml

```diff
@@ -123,7 +123,7 @@ workflows:
   scheduled-workflow:
     triggers:
       - schedule:
-          cron: "0 0,2 * * *"
+          cron: "0 0,12,15,18,21 * * *"
```

Comment:
Just wondering, why this change?

Reply:
The integration tests job runs at certain times. This should allow it to run more throughout the development day instead of just at 0:00 UTC time.